packages/client-sdk-nodejs/src/internal/vector-index-control-client.ts

```diff
@@ -1,17 +1,16 @@
 import {control} from '@gomomento/generated-types';
-import grpcControl = control.control_client;
 import {Header, HeaderInterceptorProvider} from './grpc/headers-interceptor';
 import {ClientTimeoutInterceptor} from './grpc/client-timeout-interceptor';
 import {Status} from '@grpc/grpc-js/build/src/constants';
 import {cacheServiceErrorMapper} from '../errors/cache-service-error-mapper';
 import {ChannelCredentials, Interceptor} from '@grpc/grpc-js';
 import {
-  ListCaches,
   CreateSigningKey,
-  ListSigningKeys,
-  RevokeSigningKey,
   CredentialProvider,
+  ListCaches,
```

Comment:
Shouldn't this be ListIndexes? Edit: I see below in this file that the call to `listIndexes` returns a `ListCaches`?